mountpoint-s3/CHANGELOG.md

```diff
@@ -1,4 +1,16 @@
-## Unreleased
+## v1.0.2 (September 22, 2023)
+
+### Breaking changes
+* No breaking changes.
+
+### Other changes
+* Building the Mountpoint release on CentOS 7. This gets us compatibilty back to glibc 2.17. This should let mountpoint run on OS which by default packages GCC and Clang that were too old to build CRT and run bindgen respectively in previous release. ([#517](https://github.com/awslabs/mountpoint-s3/pull/517))
```

Comment:
```suggestion
* New Mountpoint releases are now built on CentOS 7 instead of Amazon Linux 2. This lowers minimum requirement to run Mountpoint to require glibc 2.17 or newer. ([#517](https://github.com/awslabs/mountpoint-s3/pull/517))
```

Reply:
nits: 
* remove "now"
* requirement/to require sounds redundant

How about:
```suggestion
* New Mountpoint releases are built on CentOS 7 instead of Amazon Linux 2. This lowers the minimum requirement to run Mountpoint to glibc 2.17 or newer. ([#517](https://github.com/awslabs/mountpoint-s3/pull/517))
```